Anlab.Mvc/Client/components/numberInput/numberInput.tsx

```diff
@@ -29,7 +29,13 @@ export class NumberInput extends React.Component<INumberInputProps, INumberInput
         return value ? String(value) : '';
     }
     onChange = (v: string) => {
-        const error = isNaN(Number(v)) ? 'Must be a positive number' : null;
+        let error = isNaN(Number(v)) ? 'Must be a positive number' : null;
+        if (error == null) {
+            v = Number(v).toFixed(0).toString();
```

Comment:
@jSylvestre try not to ever reassign a parameter. Best option would be do a `let value = v;` at the top and then change that if necessary.